src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const set = new Set(values);
+
+  if (values.length !== set.size) {
+    return null;
+  }
```

Comment:
It's good that you already know how to use Set construction 👍 
But let's make this simple task without it and using in it more simple operations.